docs/content/docs/usage/sitemap.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to use generate sitemap in Phenomic
+---
+
+The ``phenomicLoader`` allows you to generate sitemap very easily.
+
+The phenomic-theme-base already handles a default RSS sitemap that will grab all your urls
```

Comment:
According to [sitemap's README](https://npm.im/sitemap), it generates a XML sitemap, not a RSS one.